History.md

```diff
@@ -0,0 +1,133 @@
+
+v3.0.0 / 2017-07-27
+===================
+
+  * Flush on first message (#113)
```

Comment:
I don't think the change log file should just be a commit log. It should be more detailed like https://github.com/segmentio/analytics-android/blob/master/CHANGELOG.md.

As for why having it in a file instead of GitHub releases?
1. Less steps to maintain. You edit it during a release, instead of going to the GitHub UI and pasting in the text manually.
2. Easy to search older releases. Grep etc.
3. Version controlled.

Lastly, it gives us the ability to keep a rolling log. The developer making the release doesn't always know details about every change. So as part of each commit/PR, contributors should add a section documenting the public facing version of the change, e.g. migration steps etc. Though this isn't something we're doing diligently at the moment.